test/integration-test/test-suite/src/test/resources/env/scenario/rdl_empty_rules/data/actual/init-sql/mysql/01-actual-init.sql

```diff
@@ -32,8 +32,32 @@ DROP TABLE IF EXISTS rdl_ds_0.t_user_2;
 DROP TABLE IF EXISTS rdl_ds_0.t_user_3;
 DROP TABLE IF EXISTS rdl_ds_0.t_user_4;
 
+DROP TABLE IF EXISTS rdl_ds_0.t_user_item_0;
+DROP TABLE IF EXISTS rdl_ds_0.t_user_item_1;
+DROP TABLE IF EXISTS rdl_ds_0.t_user_item_2;
+DROP TABLE IF EXISTS rdl_ds_0.t_user_item_3;
+DROP TABLE IF EXISTS rdl_ds_0.t_user_item_4;
+
+DROP TABLE IF EXISTS rdl_ds_2.t_product_item_0;
+DROP TABLE IF EXISTS rdl_ds_2.t_product_item_1;
+
+DROP TABLE IF EXISTS rdl_ds_2.t_product_0;
+DROP TABLE IF EXISTS rdl_ds_2.t_product_1;
+
 CREATE TABLE rdl_ds_0.t_user_0 (user_id INT NOT NULL, username VARCHAR(20) NOT NULL, phone VARCHAR(20) NULL, PRIMARY KEY (user_id));
 CREATE TABLE rdl_ds_0.t_user_1 (user_id INT NOT NULL, username VARCHAR(20) NOT NULL, phone VARCHAR(20) NULL, PRIMARY KEY (user_id));
 CREATE TABLE rdl_ds_0.t_user_2 (user_id INT NOT NULL, username VARCHAR(20) NOT NULL, phone VARCHAR(20) NULL, PRIMARY KEY (user_id));
 CREATE TABLE rdl_ds_0.t_user_3 (user_id INT NOT NULL, username VARCHAR(20) NOT NULL, phone VARCHAR(20) NULL, PRIMARY KEY (user_id));
 CREATE TABLE rdl_ds_0.t_user_4 (user_id INT NOT NULL, username VARCHAR(20) NOT NULL, phone VARCHAR(20) NULL, PRIMARY KEY (user_id));
+
+CREATE TABLE rdl_ds_0.t_user_item_0 (item_id INT NOT NULL, user_id INT NOT NULL, count INT NOT NULL, PRIMARY KEY (item_id));
```

Comment:
Can you use standard table in issue #21286?

Reply:
have refactor to use standard table